src/components/accordion/accordion-test.stories.mdx

```diff
@@ -0,0 +1,111 @@
+import { useState } from "react";
+import { Meta, Story, Preview } from "@storybook/addon-docs";
+import { action } from "@storybook/addon-actions";
+
+import Accordion from "./accordion.component";
+import AccordionGroup from "./accordion-group/accordion-group.component";
+import Textbox from "../textbox";
+import Box from "../box";
+
+<Meta
+  title="Design System/Accordion/Test"
+  parameters={{
+    info: { disable: true },
+    chromatic: {
+      disable: true,
+    },
+  }}
+  argTypes={{
+    iconAlign: {
+      options: ["left", "right"],
+      control: {
+        type: "select",
+      },
+    },
+    iconType: {
+      options: ["chevron_down", "dropdown"],
+      control: {
+        type: "select",
+      },
+    },
+    borders: {
+      options: ["default", "full"],
+      control: {
+        type: "select",
+      },
+    },
+    size: {
+      options: ["large", "small"],
+      control: {
+        type: "select",
+      },
+    },
+    scheme: {
+      options: ["white", "transparent"],
+      control: {
+        type: "select",
+      },
+    },
+  }}
+/>
+
+export const AccordionStory = (args) => {
```

Comment:
Do these need to be exported?

Reply:
yes because it's mdx

Reply:
Why would that mean they need to be exported? They're only being used in this file as far as I can see

Reply:
If I don't export it can't be used.
<img width="670" alt="Zrzut ekranu 2021-08-19 o 09 56 03" src="https://user-images.githubusercontent.com/77274590/130030350-fa579f80-c450-4311-9d88-abea2c89a232.png">
I think in mdx files all constants/variables have to be exported.

Reply:
So they do, I didn't know about that! 

![image](https://user-images.githubusercontent.com/14963680/130235623-e5bad5a3-3da8-4c5e-92cb-570d4c3b43b2.png)